app/controllers/summarys/shop_controller.rb

```diff
@@ -1,8 +1,10 @@
 class Summarys::ShopController < SummarysController
   def index
     @selected_app = summary_params[:selected_app]
-    @summaries = Summary::Shop.new(user: current_user, selected_app: @selected_app).summarize
+    @page = params[:page] || 1
+    @per_page = Summary::Shop::TOP_SHOPS_LIMIT
+    @summaries = Summary::Shop.new(user: current_user, selected_app: @selected_app).summarize(page: @page.to_i, per_page: @per_page.to_i)
 
-    render "summarys/index"
+    render 'summarys/index'
```

Comment:
**[standard]** <Style/StringLiterals> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
```suggestion
    render "summarys/index"
```